src/app/external-login-review-account-info-page/review-account-info/review-account-info.component.ts

```diff
@@ -0,0 +1,323 @@
+import {
+  NgFor,
+  NgIf,
+  TitleCasePipe,
+} from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Inject,
+  Input,
+  OnDestroy,
+  OnInit,
+} from '@angular/core';
+import { Router } from '@angular/router';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import {
+  TranslateModule,
+  TranslateService,
+} from '@ngx-translate/core';
+import { UiSwitchModule } from 'ngx-ui-switch';
+import {
+  combineLatest,
+  filter,
+  from,
+  map,
+  Observable,
+  Subscription,
+  switchMap,
+  take,
+  tap,
+} from 'rxjs';
+
+import { AuthService } from '../../core/auth/auth.service';
+import { AuthRegistrationType } from '../../core/auth/models/auth.registration-type';
+import { RemoteData } from '../../core/data/remote-data';
+import { EPersonDataService } from '../../core/eperson/eperson-data.service';
+import { EPerson } from '../../core/eperson/models/eperson.model';
+import { HardRedirectService } from '../../core/services/hard-redirect.service';
+import {
+  NativeWindowRef,
+  NativeWindowService,
+} from '../../core/services/window.service';
+import { Registration } from '../../core/shared/registration.model';
+import { ExternalLoginService } from '../../external-log-in/services/external-login.service';
+import { AlertComponent } from '../../shared/alert/alert.component';
+import { AlertType } from '../../shared/alert/alert-type';
+import { ConfirmationModalComponent } from '../../shared/confirmation-modal/confirmation-modal.component';
+import { hasValue } from '../../shared/empty.util';
+import { NotificationsService } from '../../shared/notifications/notifications.service';
+import { CompareValuesPipe } from '../helpers/compare-values.pipe';
+
+export interface ReviewAccountInfoData {
+  label: string;
+  currentValue: string;
+  receivedValue: string;
+  overrideValue: boolean;
+  identifier: string;
+}
+
+@Component({
+  selector: 'ds-review-account-info',
+  templateUrl: './review-account-info.component.html',
+  styleUrls: ['./review-account-info.component.scss'],
+  imports: [
+    CompareValuesPipe,
+    NgFor,
+    NgIf,
+    TitleCasePipe,
+    TranslateModule,
+    AlertComponent,
+    UiSwitchModule,
+  ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  standalone: true,
+})
+export class ReviewAccountInfoComponent implements OnInit, OnDestroy {
+  /**
+   * The AlertType enumeration for access in the component's template
+   * @type {AlertType}
+   */
+  public AlertTypeEnum = AlertType;
+  /**
+   * The registration token sent from validation link
+   */
+  @Input() registrationToken: string;
+  /**
+   * User data from the registration token
+   */
+  @Input() registrationData: Registration;
+
+  /**
+   * Text to display when the value is not applicable
+   */
+  notApplicableText = 'N/A';
```

Comment:
We may want to make this an i18n key, as I don't know that "N/A" means the same thing in all languages.